### DISCUSSION TITLE
How to add/create comment.

### DISCUSSION THREAD

**Discussion by chacan** [docmost/docmost#370] [Help / Questions] [answered]
Hello.
hi i am new to docmost, and this app is really powerful. my question is, when creating a page, can't make comments?
![image](https://github.com/user-attachments/assets/77a5df33-99b7-45bf-b011-e1af15b6899c)

**Comment by chacan** [comment, marked as answer]
thanks, with highlight.

**Reply by ja49619** [reply]
HOW to add? What is answer?
cant find

**Reply by Philipinho** [reply]
@ja49619 when you highlight/select text on the editor, you should see the comment icon in the bubble menu.

**Reply by ja49619** [reply]
found
